### PR TITLE
NATIVE-221: Upgrade to RN 0.60.6

### PR DESCRIPTION
**Pull request by klinzo**

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -11,18 +11,26 @@
 ; Ignore duplicate module providers
```
Comment (Schedulaar) [inline .flowconfig]: Can you add a link where you got the changes from or did you manually change it? :)

Comment (klinzo) [reply .flowconfig]: https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.6

**Comment by klinzo** [comment]
Fixed warnings for `rn-fetch-blob` and `react-native-vector-icons`. Created an issue for `react-native-sentry` since an upgrade includes breaking changes. Fixed linking.

**Comment by klinzo** [comment]
Regarding the warnings `componentWillReceiveProps` I'd just wait if these are resolved after upgrading all packages.

File: android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java [hunk 2]
```diff
@@ -1,28 +1,14 @@
 package tuerantuer.app.integreat;
 
 import android.app.Application;
-
-import com.RNFetchBlob.RNFetchBlobPackage;
-import com.dylanvann.fastimage.FastImageViewPackage;
+import cl.json.ShareApplication;
```
Comment (maxammann) [inline android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java]: Unused import

File: ios/Podfile [hunk 3]
```diff
@@ -1,13 +1,39 @@
...
+  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
+  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
+  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
+  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+
+  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+  pod 'rn-fetch-blob', :path => '../node_modules/rn-fetch-blob'
```
Comment (maxammann) [inline ios/Podfile]: Probably not needed

File: android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java (hunk 2 above)
Comment (klinzo) [reply android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java]: Is used, MainApplication implements it

**Pull request merged**